tools/cldr/cldr-to-icu/src/main/java/org/unicode/icu/tool/cldrtoicu/ant/CleanOutputDirectoryTask.java

```diff
@@ -340,4 +338,77 @@ private static ImmutableList<String> readLinesFromResource(String name) {
             throw new RuntimeException("cannot read resource: " + name, e);
         }
     }
+
+    private static Retain getRetain(Element elem) {
+        if (!"retain".equals(elem.getTagName())) {
+            return null;
+        }
+        String path = elem.getAttribute("path");
+        Retain retain = new Retain();
+        retain.setPath(path);
+        return retain;
+    }
+
+    private static Dir getDirectory(Element element) {
+        if (!"dir".equals(element.getTagName())) {
+            return null;
+        }
+        String name = element.getAttribute("name");
+        Dir dir = new Dir();
+        dir.setName(name);
+        Node node = element.getFirstChild();
+        while (node != null) {
+            if (node.getNodeType() == Node.ELEMENT_NODE) {
+                Element childElement = (Element) node;
+                switch (childElement.getTagName()) {
+                    case "retain":
+                        Retain retain = getRetain(childElement);
+                        dir.addConfiguredRetain(retain);
+                        break;
+                    default:
+                }
+            }
+            node = node.getNextSibling();
+        }
+        return dir;
+    }
+
+    public static CleanOutputDirectoryTask fromXml(String fileName) {
+        try {
+            DocumentBuilder builder = DocumentBuilderFactory.newInstance().newDocumentBuilder();
+            Document doc = builder.parse(new File(fileName));
+            Element root = doc.getDocumentElement();
+            if (!"config".equals(root.getTagName())) {
+                System.err.println("The root of the config file should be <config>");
+                return null;
+            }
+
+            NodeList outputDirectories = root.getElementsByTagName("outputDirectories");
+            if (outputDirectories.getLength() != 1) {
+                System.err.println("Only one <outputDirectories> element allowed and required");
```

Comment:
```suggestion
                System.err.println("Exactly one <outputDirectories> element allowed and required");
```

Reply:
Accepted. Done.

Reply:
this one is not done

Reply:
Weird. I see it locally, and here:
https://github.com/unicode-org/icu/pull/3290/commits/67705cc76fbeb939ceebfbc831a2bfa7dd06dbca

Reply:
Ah, same fix in two files.
My bad.